docroot/lib/CgovPlatform/Tests/CgovSchemaExclusions.php

```diff
@@ -39,6 +39,10 @@ class CgovSchemaExclusions {
     // for the tracked issue.
     'adobe_dtm.settings',
     'views.view.samlauth_map',
+    'views.view.content',
```

Comment:
Can you please add a comment for the specific error that you were getting? I forget what it was exactly, but it would be good if someone hits it in the future. (It was something to do with missing schema for settings for dates which should have broken for other uses...)